src/include/ipc/header.h

```diff
@@ -331,6 +331,8 @@ struct ipc_cmd_hdr;
 /** Maximum message size for mailbox Tx/Rx */
 #if CONFIG_IPC_MAJOR_4
 #define SOF_IPC_MSG_MAX_SIZE			0x1000
+#elif CONFIG_LIBRARY_STATIC || UNIT_TEST
+#define SOF_IPC_MSG_MAX_SIZE			0x2000
```

Comment:
How much do we care about the unit tests and libraries actually test IPC? If we care a lot then we should make this at best temporary.

Reply:
Unit tests and testbench bypass most of the IPC code. The SOF plugin and  testbench IPC version (still a draft PR) are using the IPC library code so they would add coverage once they are ready for usage. In short term for unit tests and testbench I don't want to care much about IPC coverage but with the new frameworks I hope it gets better.